Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -134,6 +134,36 @@ extension BrowserViewController {
         Preferences.NewTabPage.atleastOneNTPNotificationWasShowed.value = false
     }
     
+    static func migrateAdsConfirmations(for configruation: BraveRewardsConfiguration) {
+        // To ensure after a user launches 1.21 that their ads confirmations, viewed count and
+        // estimated payout remain correct.
+        //
+        // This hack is unfortunately neccessary due to a missed migration path when moving
```

Comment:
There is currently no way this logic can move to the ads lib as Rewards has now deprecated save/load functionality so there would be no way to access the file

Reply:
Right, just meant it should have been handled in brave-core where these directories & paths are known, but due to timing we will add this change here with hard-coded paths & file names